.eslintrc

```diff
@@ -69,14 +69,13 @@
         ]
       }
     ],
-    "sort-imports": ["error", { "ignoreDeclarationSort": true }],
-    "import/order": ["error", { "alphabetize": { "order": "asc" } }],
     "import/no-useless-path-segments": [
       "warn",
       {
         "noUselessIndex": true
       }
-    ]
+    ],
+    "simple-import-sort/imports": "error"
```

Comment:
I wonder why we have marked this as `warn` in the components repo and as `error` in this one.

[Ref](https://github.com/cloudscape-design/components/blob/ae7e61f6ebbfde4b1673e852d8773ff7072bef31/.eslintrc.js#L91)

Reply:
This rule has already been promoted to `error` in the components repo: https://github.com/cloudscape-design/components/blob/main/.eslintrc.js#L91